Neos.Cache/Classes/Backend/PdoBackend.php

```diff
@@ -464,4 +506,134 @@ protected function context(): string
         }
         return $this->context;
     }
+
+    /**
+     * TODO document
+     *
+     * @return Result
+     * @api
+     */
+    public function getStatus(): Result
```

Comment:
Thinking again about this, what if we instead of adding all those `BackendWithSetupInterface` and `BackendWithStatusInterface` responsibilities, we go another route and solve this by composition instead? So build a `PdoBackendSetup` class that receives the `PdoBackend` instance and then does the setup for this backend - it only needs access to the table names and the databaseHandle. This way we could clearly separate the resposibilities for setting up and getting the status (monitoring), instead of loading them onto the sometimes already big Backend implementation classes and increasing the list of interfaces a backend implements. WDYT?

Reply:
Usually I'm always in favor of composition. In this case it wouldn't replace inheritance though but just add another abstraction. And in this case I think it wouldn't improve things really because IMO it's the responsibility of the backend implementation to know about its status and the code needs to know a lot of the inner workings per definition.

Reply:
I was more coming from the Single Responsibility Principle side of things. Setting itself up and watching it's status is not the primary responsibility of a Cache Backend. It was just an idea though on how to aproach this in a way, that will prevent adding another ~200 LOCs to an already rather complex backend implementation.